src/test/components/AppLocalizationProvider.test.js

```diff
@@ -66,4 +89,22 @@ describe('AppLocalizationProvider', () => {
       getByTestId('content-test-AppLocalizationProvider')
     ).toBeInTheDocument();
   });
+
+  it('Fetches the en-US FTL strings and renders them', async () => {
```

Comment:
nit: it would be good to make this lower case "f" because during testing, all the `describe` and `it` blocks are concatenated together. It's good to form a sentence when `describe` + `it` blocks are together.

For example for this test: 
Describe block: "AppLocalizationProvider"
it block is: "fetches the en-US FTL strings and renders them"

So the result becomes:
"AppLocalizationProvider › fetches the en-US FTL strings and renders them" or
"AppLocalizationProvider fetches the en-US FTL strings and renders them"

Reply:
Oh thanks, I didn't know that. I've changed it. Also changed the upper `it` block too.